maisim/maisim.Game/Component/Gameplay/Notes/DrawableTouchNote.cs

```diff
@@ -66,6 +66,11 @@ protected override Drawable[] AddNoteParts(TextureStore textureStore)
             };
         }
 
+        public override string GetEncodeString()
```

Comment:
This feels a bit fragile, may be better to design an abstraction over this to handle potential pain points.